index.html

```diff
@@ -1 +1,64 @@
-GitHub Pages placeholder
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>Tubes</title>
+
+    <!-- Bootstrap core CSS -->
+    <link href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
// for stylesheets/etc will use whatever protocol the page is, meaning you won't get mixed content errors when loading it.